hash/generic_cdrom.xml

```diff
@@ -169,12 +174,1004 @@ pc98: access [PC-9801-73] registers
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="winnt40ts" cloneof="winnt40">
+		<description>Windows NT Server, Terminal Server Edition 4.0 (4.0.419.1)</description>
+		<year>1998</year>
+		<publisher>Microsoft</publisher>
+		<sharedfeat name="compatibility" value="alpha,ibmpc" />
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="en-us windows nt terminal server 4.0.419.1" sha1="31b87bdf7ad173ba1643c2b11acb08a45fa17ae1"/>
+			</diskarea>
+		</part>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 1"/>
+			<dataarea name="flop" size="1474560">
+				<rom name="en-us windows nt terminal server 4.0.419.1 3.5-1.img" size="1474560" crc="d49a1246" sha1="b23acea288d91b0ef4b40ed0f68a336561de2ad1"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 2"/>
+			<dataarea name="flop" size="1474560">
+				<rom name="en-us windows nt terminal server 4.0.419.1 3.5-2.img" size="1474560" crc="61b8e332" sha1="bfe47d04d39bba2ae53afab83d6a939be624ed40"/>
+			</dataarea>
+		</part>
+		<part name="flop3" interface="floppy_3_5">
+			<feature name="part_id" value="Disk 3"/>
+			<dataarea name="flop" size="1474560">
+				<rom name="en-us windows nt terminal server 4.0.419.1 3.5-3.img" size="1474560" crc="89fd5fb4" sha1="1bd3648bbe1c081d2694b1450fed5dbd8e056d02"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="winnt40e" cloneof="winnt40">
+		<description>Windows NT Server, Enterprise Edition 4.0 (4.0.1381.4)</description>
+		<year>1997</year>
+		<publisher>Microsoft</publisher>
+		<notes><![CDATA[
+		This installs as 4.0.1381.2 (SP1), but immediately prompts to install
+		Service Pack 3 to bring it to 4.0.1381.4. Let's go with that number.
+		]]></notes>
+		<sharedfeat name="compatibility" value="alpha,ibmpc" />
+		<part name="cdrom1" interface="cdrom">
+			<feature name="part_id" value="Operating System"/>
+			<diskarea name="cdrom">
+				<disk name="en-us windows nt enterprise server 4.0.1381.4 disc 1" sha1="35f1693e9f40dee0d9019a0db3dbaa01812d596d"/>
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<feature name="part_id" value="Components"/>
+			<diskarea name="cdrom">
+				<disk name="en-us windows nt enterprise server 4.0.1381.4 disc 2" sha1="80e4aebb1808a1c3ea43b2f401874612d0ff0394"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Did Service Pack 1 ever have a CD-ROM release? -->
+	<software name="winnt40sp2">
+		<description>Windows NT 4.0 Service Pack 2</description>
+		<year>1996</year>
+		<publisher>Microsoft</publisher>
+		<info name="release" value="19961213" />
+		<info name="language" value="English" />
+		<info name="usage" value="This updates the operating system, but is not the operating system installer itself." />
+		<sharedfeat name="compatibility" value="alpha,ibmpc,prep" />
```

Comment:
I guess I'll apply then modify by myself, and I need two pieces of information:
1. what `alpha` exactly stands for? Does a driver exists for it?
2. what `prep` stands for?